packages/story-editor/src/components/library/panes/media/media3p/test/media3pPane.js

```diff
@@ -241,6 +253,7 @@ describe('Media3pPane', () => {
   it('should render <Media3pPane /> with the category display name when selected', () => {
     useMediaResult.media3p.PROVIDER_1.state.isMediaLoaded = true;
     useMediaResult.media3p.PROVIDER_1.state.media = MEDIA;
+    useMediaResult.searchTerm = '';
```

Comment:
I just copied over the patterns used in the rest of this test. Because I set `useMediaResult.searchTerm = 'cats';` I had to reset it to an empty string here. I had thought that the 
```beforeEach(() => {
    useMediaResult = DEFAULT_USE_MEDIA_RESULT;
  });
```
on line 148 would have reset it to be empty. BUT that doesn't seem to be the case.

Reply:
Let me know if y'all think I should make this better. 🤷🏻‍♀️

Reply:
think it's fine as is!